README.md

```diff
@@ -89,6 +89,10 @@ vendor/bin/rector process ../project --autoload-file ../project/vendor/autoload.
 
 Or make use of `rector.yaml` config:
 
+```bash
+vendor/bin/rector process ../project --config ../project/rector.yaml
+```
```

Comment:
The line above refers about the confit content, not `--config` option.

Better add use case bellow with example, when this option useful